Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-buster as base
+
```

Comment:
You could also include some environment variables that won't need to vary, e.g. `ENV FLASK_ENV=production` for the production stage or =development for development.

This is currently covered by your .env file but this change would save a small amount of config when setting up other environments.